easybuild/easyblocks/q/qt.py

```diff
@@ -151,6 +153,12 @@ def configure_step(self):
         ]
         run_cmd_qa(cmd, qa, no_qa=no_qa, log_all=True, simple=True, maxhits=120)
 
+        # Ninja uses all visible cores by default, which can lead to lack of sufficient memory;
+        # so $NINJAFLAGS is set to control number of parallel processes used by Ninja
+        if LooseVersion(self.version) >= LooseVersion('5'):
+            if get_software_root('Ninja'):
+                env.setvar('NINJAFLAGS', '-j%s' % self.cfg['parallel'])
```

Comment:
It's likely a bad idea to overwrite NINJAFLAGS like this. User could have set it for some reason. And it should check if "-j" is already in NINJAFLAGS and not override it if it is...

Reply:
Should one add this variable on ninja itself? Or make ninja respect parallel?

Reply:
I've just figured out that `$NINJAFLAGS` is very specific to `Qt5`, so in that case I'd argue that ignoring `$NINJAFLAGS` if it's set already actually makes total sense (you don't want a random value for a pretty generically named environment variable like `$NINJAFLAGS` to mess up your Qt5 build).